Packs/AzureStorage/Integrations/AzureStorage/AzureStorage.yml

```diff
@@ -0,0 +1,220 @@
+category: IT Services
+commonfields:
+  id: Azure Storage
+  version: -1
+configuration:
+- defaultvalue: ab217a43-e09b-4f80-ae93-482fc7a3d1a3
+  display: Application ID
+  hidden: false
+  name: app_id
+  required: true
+  type: 0
+- display: Subscription ID
+  hidden: false
+  name: subscription_id
+  required: true
+  type: 0
+- display: Resource Group Name
+  hidden: false
+  name: resource_group_name
+  required: true
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Deploy and manage containerized applications with a fully managed Kubernetes
+  service.
+display: Azure Storage (Beta)
+name: Azure Storage
+script:
+  commands:
+  - deprecated: false
+    description: Tests the connectivity to Azure.
+    execution: false
+    name: azure-storage-auth-test
+  - deprecated: false
+    description: Run this command to start the authorization process and follow the
+      instructions in the command results.
+    execution: false
+    name: azure-storage-auth-start
+  - deprecated: false
+    description: Run this command to complete the authorization process. Should be
+      used after running the azure-storage-auth-start command.
+    execution: false
+    name: azure-storage-auth-complete
+  - deprecated: false
+    description: Run this command if for some reason you need to rerun the authentication
+      process.
+    execution: false
+    name: azure-storage-auth-reset
+  - arguments:
+    - default: false
+      isArray: false
+      name: account_name
+      required: false
+      secret: false
+    deprecated: false
+    execution: false
+    name: azure-storage-account-list
```

Comment:
I think you are missing command, argument and context descriptions - please add them in all the yml (can be done through the demisto-settings section in the pycharm plugin.